test/Java8andUp/src/org/openj9/test/java/security/Test_AccessController.java

```diff
@@ -494,17 +493,6 @@ public PermissionCollection getPermissions(CodeSource cs) {
 				Object ret = AccessController.doPrivileged(new PrivilegedAction<Object>() {
 					public Object run() {
 						try {
-							/* This tests a use scenario with following stacktrace from a standalone testcase
```

Comment:
Why is this removed?

Reply:
I took out the stack traces because they clutter the code and don't add much value.
They also may not be accurate still.

Reply:
They add value when understanding what the tests are doing, and when having to debug a failure. Please leave them in.

Reply:
Done.